### ISSUE TITLE
Javascript Standard Style fixes are outdated and can't be applied to the document

### ISSUE THREAD

**Issue by Jakobud** [standard/vscode-standard#6]
I get an info box with this when trying to autofix my JavaScript. Not sure what I'm doing wrong or what this means exactly.

**Comment by Jakobud** [comment]
I have a JavaScript document like this:

```
const services = {
  'myobject': {
    'test': [{
      'hello': 'world'
    },
    {
      placeHolder: 'hello',
      action: 'input',
      regex: '^.+$'
    }]
  }
}
```

I use `cmd + P` and run `JavaScript Standard Style: Fix all auto-fixable problems`. Then the above code turns to this:

```
const services = {
  'myobject': {
    'test': [{
      'hello': 'world'
    },
      {
        placeHolder: 'hello',
        action: 'input',
        regex: '^.+$'
      }]
  }
}
```

And I get the infobox saying:

```
Javascript Standard Style fixes are outdated and can't be applied to the document
```

If I run the `fix` command a second time, the code now turns to this:

```
const services = {
  'myobject': {
    'test': [{
      'hello': 'world'
    },
                    placeHolder: 'hell              action: 'in                regex:      +$'
      }]
  }
}
```

Seems like this extension is broken or something. Am I doing something wrong?

**Comment by chenxsan** [comment]
Are you running vscode-standard-format instead of vscode-standardjs? Because you are running command `JavaScript Standard Format: Fix all auto-fixable problems`.

In vscode-standardjs we have this command :

<img width="609" alt="screen shot 2016-12-29 at 11 39 01 am" src="https://cloud.githubusercontent.com/assets/1091472/21536266/8787fe08-cdbb-11e6-858a-5d0f8ec718fd.png">

**Comment by Jakobud** [comment]
Sorry for the confusion. The command does say "Style" not "Format". I will fix my above comments. The extension I have installed is JavaScript Standard Style 1.0.8.

**Comment by chenxsan** [comment]
This is weird, since there no such `can't be applied to the document` in this extension code. You can check it in the parent repository from https://github.com/Microsoft/vscode-eslint/search?q=can't+be+applied+to+the+document&type=Issues&utf8=✓

Maybe there's other extension trying to edit the file too and making the StandardJS fix outdated. Would you please try it with only this extension installed in VSCode?

**Comment by Jakobud** [comment]
Okay I'll look into this. What exactly does this extension do beyond the original `vscode-eslint` extension?

Also here is a screenshot of the infobox in question:

<img width="663" alt="screen shot 2016-12-29 at 8 34 39 pm" src="https://cloud.githubusercontent.com/assets/87356/21559737/6d8f1bc6-ce06-11e6-9313-cdd185993209.png">

**Comment by chenxsan** [comment]
It's almost same as `vscode-eslint` but with some [predefined rules](http://standardjs.com/rules.html). In `vscode-eslint`, I think you need to define your own rules.

**Comment by chenxsan** [comment]
Seems to be fixed in 1.0.9.
